features/prompt_memes_b/challenge_promptmeme_posting_fills.feature

```diff
@@ -39,9 +41,7 @@ Feature: Prompt Meme Challenge
   Then I should see "Kinky Story"
     And I should find a list for associations
     And I should see "In response to a prompt by Anonymous in the promptcollection collection"
-  # TODO: Figure out why this isn't working
-  # email the anonymous prompter that they've received a fill!
-   # And 1 email should be delivered to "my1@e.org"
+    And 1 email should be delivered to "my1@e.org"
```

Comment:
Drat -- I was aiming for this line and missed: https://coveralls.io/builds/11242399/source?filename=app%2Fmodels%2Fcreation_observer.rb#L115 But the test is fixed anyway. Yay?